src/client/elm/Chain/Build.elm

```diff
@@ -34,15 +34,16 @@ type alias Block =
     , nodesAt : List String
     , fractionNodesAt : Float
     , status : BlockStatus
-    , forkWidth : Int
-    , blockHeight : Int
+    , x : Int
+    , y : Int
```

Comment:
Yes i understand your point. I just found block height particularly confusing in this context because it is not visualised as height. But ok, if we keep `blockHeight`, maybe we call `y` `branchPosition` or maybe `branchIndex`?

Reply:
`branchPosition` is fine with me